check_vm_backup.ps1

```diff
@@ -18,17 +19,32 @@ if($vm -eq $FALSE) {
     exit 3
 }
 
-$now=[Math]::Floor([decimal](Get-Date(Get-Date).ToUniversalTime()-uformat "%s"))
+if($vm -eq $FALSE) {
+    echo "param -type is missing and must either be Backup or Copy. Leave it empty for Backupjobs"
+    exit 3
+}
 
-foreach($job in (Get-VBRJob)) {
+if($type -eq "Copy") {
+    $type="BackupSync"
+}
+
+if($type -ne "Backup" -and $type -ne "Backupsync") {
+    echo "parameter -type wrong, please check the spelling"
```

Comment:
echo "parameter -type accepts backup or copy as value"

Reply:
changed to the comment suggested